### ISSUE TITLE
[julia/en] Changes to typing system

### ISSUE THREAD

**Issue by ecarl65** [adambard/learnxinyminutes-docs#3029]
Playing with the latest version of Julia (0.6.2), there are some changes to the typing system.

The tutorial at one point says:
``` julia
# abstract Name
abstract Cat # just a name and point in the type hierarchy
```

But my Julia repl says it should be:
``` julia
WARNING: deprecated syntax "abstract Cat" at REPL[18]:1.
Use "abstract type Cat end" instead.
```

Some of the abstract types may have changed, when I look for the subtypes for the AbstractString I get different results from the comments.
``` julia
julia> subtypes(AbstractString)    # 8-element Array{Any,1}:
6-element Array{Union{DataType, UnionAll},1}:
 Base.SubstitutionString
 Base.Test.GenericString
 DirectIndexString      
 RevString              
 String                 
 SubString 
```

Also, in this version of Julia it says that `super` isn't defined:
``` julia
julia> super(Int64) # => Signed
ERROR: UndefVarError: super not defined
```

I haven't done a pull request on other's projects before, but perhaps I'll make these fixes myself and submit the request, referencing this issue.

**Comment by heitorPB** [comment]
I think those are solved in #2780, but it is not merged yet.

**Comment by divayprakash** [comment]
Julia tutorial has been updated to comply with v1.0
